requirements.txt

```diff
@@ -31,13 +31,13 @@ httplib2shim>=0.0.3
 
 # Azure Provider
 ## core requirements
-azure-cli-core==2.12.0
+azure-cli-core==2.18.0
```

Comment:
By updating these dependencies, some authentication mechanisms no longer work.

For `--user-account` authentication I'm getting:
```
2021-02-05 11:47:11 wrkbx2 scout[319] ERROR storageaccounts.py L27: Failed to retrieve storage accounts: 'UserPassCredentials' object has no attribute 'get_token'
```

Reply:
Done in another PR